omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ObsController1_9Test.java

```diff
@@ -84,6 +87,38 @@ public void shouldGetAll() throws Exception {
 	}
 	
 	@Test
+	public void shouldGetallIncludingVoidedObs() throws Exception {
```

Comment:
The test method name does not clearly state what you are testing. Can you look at this for an example? https://github.com/openmrs/openmrs-module-webservices.rest/commit/191e0b6f706af6ae9244798d6d9044114f93ccfd

Reply:
@dkayiwa 
Working on it.

Reply:
This has been corrected.